sdk/tests/conformance/extensions/ext-disjoint-timer-query.html

```diff
@@ -58,6 +62,15 @@
         testPassed("No EXT_disjoint_timer_query support -- this is legal");
     } else {
         runSanityTests();
+
+        // Clear disjoint value.
+        gl.getParameter(ext.GPU_DISJOINT_EXT);
+
+        runElapsedTimeTest();
+        runTimeStampTest();
+
+        gl.finish();
+        setTimeout(checkQueryResults, 0);
```

Comment:
This is definitely an improvement, but I'd like to request that we use `requestAnimationFrame(checkQueryResults)` here and below instead of `setTimeout`. This test will likely be a template for future query tests, and as such we should try and model it after more realistic code/best practices.